tests/gold_tests/headers/forwarded.gold

```diff
@@ -2,6 +2,8 @@ FORWARDED MISSING
 -
 FORWARDED MISSING
 -
+FORWARDED MISSING
+-
```

Comment:
This output is from an observer, which begins observing even before the tests run. The extra entry here is due to the health check.

Reply:
Yes, this was my concern about the health checks. The observer is called from the micro-server on every request, *including the health check* request. I think we will need to tweak the micro-server to be able to mark a request/response as a health check in which case it does not call the observers.